### ISSUE TITLE
Explanation of additional parameters (e.g. seq_length)

### ISSUE THREAD

**Issue by kotrfa** [karpathy/char-rnn#31]
Hello,

would it be possible to add some additional info to README about additional parameters for `train.lua`? Help under the `-help` tag is OK, but more information would be welcome. For example, I'm not really sure what `-seq_length` means from this help: `number of timesteps to unroll for`, 3 decay rates, max_epoch, grad_clip... Or what about number of epochs - if I see that since e.g 10th epoch the validation loss function doesn't decrease, can I reduce the number of epochs?

The reason why I ask is because of parameters tunning. Should I tune these also, next to dropout and size of the network (as is in README)?

**Comment by karpathy** [comment]
Hi, thanks, I didn't expect the code to get this wide reach so I didn't put a lot of effort into the descriptions. I'll try to document more thoroughly!
Andrej

**Comment by kotrfa** [comment]
Of course - it's understandable :) . Thank you (even for the answer at issue 32)

Next enhancement would be to add `-quiet` or `-verbose` switch to tell the `sample.lua` if it should print out the messages like `using CUDA` or `creating network` etc. I have already done in my version, but it doesn't even deserve a pull request :D .
